style/ruby/README.md

```diff
@@ -3,12 +3,11 @@ Ruby
 
 [Sample](sample.rb)
 
-* Avoid conditional modifiers (lines that end with conditionals).
 * Avoid multiple assignments per line (`one, two = 1, 2`).
 * Avoid organizational comments (`# Validations`).
 * Avoid ternary operators (`boolean ? true : false`). Use multi-line `if`
   instead to emphasize code branches.
-* Avoid explicit return statements.
```

Comment:
Also, I'm curious about why this was "avoid" to begin with rather than an explicit "don't". I wonder if there are cases in ruby where you actually need an explicit return (outside of guard clauses).